apps/src/studio/studio.js

```diff
@@ -3327,18 +3337,31 @@ Studio.animateClouds = function() {
   }
 
   for (var i = 0; i < constants.NUM_CLOUDS; i++) {
+    Studio.cloudStep++;
```

Comment:
Is `cloudStep` supposed to be incremented multiple times within one call to `animateClouds()`? Does any of the code elsewhere assume that there are two increments (NUM_CLOUDS) per call?